TrackingTools/TrackRefitter/interface/TrackTransformer.h

```diff
@@ -40,12 +37,22 @@ class TrajectorySmoother;
 class Propagator;
 class TransientTrackingRecHitBuilder;
 class Trajectory;
+class GlobalTrackingGeometryRecord;
+class IdealMagneticFieldRecord;
+class TrajectoryFitterRecord;
+class TrackingComponentsRecord;
+class TransientRecHitRecord;
 
 class TrackTransformer final : public TrackTransformerBase {
 public:
-  /// Constructor
+  /// Constructor (for modules not yet migrated to ES-consumes)
   explicit TrackTransformer(const edm::ParameterSet&);
```

Comment:
uhm, in the PR description `Removing the non-ESGetToken code path is not feasible at this time, because TrackTransformer is used in Alignment in an EDLooper (which, as of today, can not declare consumption of ED nor ES products #31745).`

OK